SmartDeviceLink/SDLFileManager.h

```diff
@@ -98,11 +99,19 @@ typedef void (^SDLFileManagerStartupCompletionHandler)(BOOL success, NSError *__
 /**
  *  Upload a file to the remote file system. If a file with the [SDLFile name] already exists, this will overwrite that file. If you do not want that to happen, check remoteFileNames before uploading, or change allowOverwrite to NO.
  *
- *  @param file       An SDLFile that contains metadata about the file to be sent
+ *  @param file       A SDLFile that contains metadata about the file to be sent
```

Comment:
Not to get too grammar nazi...but I'm going to anyway. This should be "An," due to "SDL" starting with a vowel sound (an "e", as in es-dee-el). The sound matters, the actual letter used does not.

Reply:
Fixed